### PR TITLE
feat: Add the launcher root folder to the Folders menu

### LINKED ISSUE #70: Add the root launcher folder to the "Folders" menu button
### Role

I play modded Minecraft

### Suggestion

Add the root launcher folder to the "Folders" menu alongside the instance folder and central mods folder.

### Benefit

Gives users direct access to an important folder.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

There are times when users need to access the root launcher folder for, say, the `accounts.json` file or the `polymc.cfg` file. Indeed, with this recent drama, users have needed to access this folder and have needed to [follow instructions](https://gist.github.com/Earthcomputer/dc65391f84a2c19ebac6c33506fd7751) to get to it. But the path for Linux doesn't work for Flatpak installs.

I personally click the "Instance Folder" button in the sidebar then go up two folders. But I can only rely on this because I haven't set the instance folder to something else.

There should just be a link somewhere to the root folder.

### PR DESCRIPTION
**Pull request by RedsonBr140**
There it is, closes #70:

https://user-images.githubusercontent.com/61910521/236957076-b07d1667-dca8-42a1-9da0-5e1ab19442e2.mp4

I don't know if the name I choose is the best one because my english actually sucks. If you guys think there's a better name please let me know.

### CODE REVIEW

**Comment by RedsonBr140** [comment]
~~Not working with `-d` btw, I'll fix it.~~
Now it does

**Review by flowln — APPROVED** [review]

File: launcher/ui/MainWindow.cpp [hunk 1]
```diff
@@ -1201,6 +1201,14 @@ void MainWindow::on_actionViewInstanceFolder_triggered()
     DesktopServices::openDirectory(str);
 }
 
+void MainWindow::on_actionViewLauncherRootFolder_triggered()
+{
+    QDir dataDir = QDir::current();
+    QString dataPath = dataDir.absolutePath();
+
```
Comment (TheKodeToad) [inline launcher/ui/MainWindow.cpp]:
```suggestion
    const QString dataPath = QDir::currentPath();
```

**Review by TheKodeToad — APPROVED** [review]
This is fine
However, I had a look at some documentation and found something more concise which *should* work.

thank you, this is definitely a feature I'd use! :D

**Comment by Scrumplex** [comment]
I would say this is minor enough for a 7.1 release

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**